package.json

```diff
@@ -63,7 +64,7 @@
     "stylelint-scss": "^3.11.1"
   },
   "engines": {
-    "node": ">= 10.*"
+    "node": "8.* || >= 10.*"
```

Comment:
Can you revert this one? The ember template supports back to 8, but we don't need to.

Reply:
handled.